app/v2/controllers/requestParsers/validators/SubmitEndOfPeriodStatementValidator.scala

```diff
@@ -16,8 +16,17 @@
 
 package v2.controllers.requestParsers.validators
 
-import v2.controllers.requestParsers.validators.validations._
-import v2.models.errors._
+import api.controllers.requestParsers.validators.Validator
+import api.controllers.requestParsers.validators.validations.{
```

Comment:
Can be changed to: import api.controllers.requestParsers.validators.validations._